apps/theming-designer/src/components/ThemingDesigner.tsx

```diff
@@ -50,16 +50,15 @@ export class ThemingDesigner extends BaseComponent<{}, IThemingDesignerState> {
     this._onPrimaryColorPickerChange = this._onPrimaryColorPickerChange.bind(this);
     this._onTextColorPickerChange = this._onTextColorPickerChange.bind(this);
     this._onBkgColorPickerChange = this._onBkgColorPickerChange.bind(this);
+  }
 
-    hideSemanticSlots = true;
+  public render() {
+    hideSemanticSlots = false;
```

Comment:
Can probably inline this now instead of keeping as a branch.

Reply:
done.